Add Minecraft 3DS Cheat - Mob/Entity Cap Remover

Cracko298 opened the pull request:
As the title suggests, these have been tested on the lastest version of the Title.

JourneyOver commented:
Please remove the `{Created by Cracko298, removed the Cap for Mobs/Entities. Version 1.2}` bits, and add the version information to the title of cheats.

Cracko298 commented:
They where tested on all versions of the game on both real hardware and an emulator to ensure compatibility across the board.

Also fixed codes like you asked, added versioning to title and removed credits.

JourneyOver commented:
Alright - LGTM Thanks!
